library/src/main/java/com/schibsted/spain/barista/interaction/BaristaScrollInteractions.kt

```diff
@@ -22,8 +21,8 @@ object BaristaScrollInteractions {
     private val MAX_SCROLL_ATTEMPTS = 50
 
     @JvmStatic
-    fun scrollTo(@IdRes id: Int) {
-        scrollWithMultipleAttempts(withId(id), failAtEnd = true)
+    fun scrollTo(id: Int) {
```

Comment:
Maybe rename to "resId" for consistency with other methods (click), since now we're not only receiving an id.

Reply:
On the way!